proposals/jorydotcom-PROJECT_PROGRESSION/PROJECT_PROGRESSION.md

```diff
@@ -0,0 +1,116 @@
+1. Any project that realistically intends to join the _$FOUNDATION_ Incubation or Top Level Stages in the future and wishes to lay the foundations for that transition.
+
+**Expectations**
+End users should evaluate Sandbox projects with care - some Sandbox projects may fail, and others may stay in the  Sandbox Stage indefinitely. Sandbox stage projects will receive minimal marketing support from the _$FOUNDATION_. Sandbox Stage projects will be reviewed on an annual basis; they may also request a status review by submitting a report to the CPC.
+
+**Acceptance Criteria**
+To be considered for Sandbox Stage, the project must meet the following requirements:
+* 2 CPC sponsors to champion the project & provide mentorship
+* A presentation to at the meeting of the CPC
+* Adherence to the _$FOUNDATION_ IP Policy
+* Upon acceptance, Sandbox projects must list their status prominently on website/readme
+
+### Incubating Stage*
+**Definition** 
+The Incubation Stage is for projects that are interested in reaching the Top-Level Stage, or want to have more access, participation, responsibilities and representation in the CPC. Incubating projects will receive mentorship from the CPC and are expected to actively develop their community of contributors, governance, and other project documentation. 
+
+**Examples**
+1. Projects that are on their way or very likely to become Top Level Projects.
+
+**Expectations**
+Projects can remain in the Incubating stage indefinitely, but they are generally expected to graduate within two years.
+
+**Acceptance Criteria**
+To be considered for Incubation Stage, the project must meet the Sandbox requirements as well as the following:
+ * Document that it is being used successfully in production by at least three independent end users which, in the CPC’s judgement, are of adequate quality and scope.
+ * Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
+ * Demonstrate a substantial ongoing flow of commits and merged contributions.
+ * Since these metrics can vary significantly depending on the type, scope and size of a project, the CPC has final judgement over the level of activity that is adequate to meet these criteria.
+ * Recieve a two-thirds supermajority vote of the CPC to move to Incubation Stage. 
+
+
+### Top Level/Graduate Stage*
+**Definition**
+
+**Examples**
+
+**Expectations**
+
+**Acceptance Criteria**
+To graduate from Sandbox or Incubating status, or for a new project to join as a Top-Level project, a project must meet the Incubating stage criteria plus:
+
+ * Have committers from at least two organizations.
+ * Adopt the _$FOUNDATION_ Code of Conduct.
+ * Explicitly define a project governance and committer process. This preferably is laid out in a GOVERNANCE.md file and references an OWNERS.md file showing the current and emeritus committers.
```

Comment:
I don't think this is enough. It should be _open_ governance. A BFDL model is not acceptable for top level projects.

I would not reference a `OWNERS.md` file here.

Reply:
Let me be devil's advocate here:

> It should be open governance

How do we define this?

> A BFDL model is not acceptable for top level projects.

Why not?

Reply:
> How do we define this?

Open governance is a model that enables all collaborators to be part of the decision making of the project.
In [Node.js we have](https://github.com/nodejs/node/blob/master/GOVERNANCE.md#nodejs-project-governance):


> The Node.js project is governed by its Collaborators, including a Technical Steering Committee (TSC) which is responsible for high-level guidance of the project.

@jasnell wdyt?

> Why not?

I am a true believer in Open Governance for large-scale open source projects. As an example, if a BFDL model is allowed, would the BFDL sit on the CPC on behalf of their project? IMHO that creates a bad pattern/incentive for individuals in adding projects to $foundation. An elected director/chair that serves a term is ok, I'm not comfortable in the "for life" part.

A successful Open Source project survive its creator exiting the project.

Reply:
I agree it should be Open Governance to be a top level project.

Reply:
In terms of "why not" I think one of the reasons is that as a top level project you share the $ which are being contributed by the members.  As part of accepting that support the project should have governance that allows members of the foundation, and others to participate in the project in a meaningful way.